internal/proxy/task.go

```diff
@@ -1305,7 +1309,10 @@ func (lct *loadCollectionTask) PreExecute(ctx context.Context) error {
 }
 
 func (lct *loadCollectionTask) Execute(ctx context.Context) (err error) {
-	log.Debug("loadCollectionTask Execute", zap.String("role", typeutil.ProxyRole), zap.Int64("msgID", lct.Base.MsgID))
+	traceID, _, _ := trace.InfoFromContext(ctx)
+	log.Debug("loadCollectionTask Execute",
```

Comment:
use log.With() to make sure all logs here will output role, traceID and collectionID, 1313 misses collectionID